### PR TITLE
Contributing.md  updated to make it more friendly.

### PR DESCRIPTION
**Pull request by mbj36**
Document made more friendly :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,10 +3,20 @@
...
+* Make sure you have Git, Vagrant and other dependencies installed on your system.
+       * [Download Git](https://git-scm.com/downloads)
+       * [Download Vagrant](https://www.vagrantup.com/downloads-archive.html)
+       * Vagrant uses virtualBox as its provider.
+       * [Download VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+       * Install the dependencies according to your Oeprating system.
+       
+* After installing the above dependencies change your working directory which you would like to use as your project directory.
+* Run the following command on the terminal
+       * Type ```vagrantinit```
```
Comment (leto) [inline CONTRIBUTING.md]: Looks like you need a few spaces on these lines

**Comment by mbj36** [comment]
@leto 
I have removed the spaces. 
Please review it 

Thanks

**Comment by leto** [comment]
@mbj36 I meant that the command "vagrantinit" should be "vagrant init" and similarly for the next few lines

**Comment by mbj36** [comment]
@leto  
Apology for the misconception.
I have made the command right and squashed into a single commit.
Please review it 

Thanks

**Comment by leto** [comment]
Thanks @mbj36 !

**Pull request merged**